js/ui/pivot_grid/ui.pivot_grid.js

```diff
@@ -270,7 +270,14 @@ var PivotGrid = Widget.inherit({
                  * @type number
                  * @default 600
                  */
-                height: 600
+                height: 600,
+                /**
+                 * @name dxPivotGridOptions_fieldChooser_applyChangesMode
+                 * @publicName applyChangesMode
+                 * @type string
+                 * @default "instantly"
```

Comment:
Accepted values are missed.
It is better using an enum instead of string type in this case.